onnx/shape_inference.py

```diff
@@ -88,18 +88,30 @@ def infer_node_outputs(
     input_types: Dict[str, onnx.TypeProto],
     input_data: Optional[Dict[str, onnx.TensorProto]] = None,
     input_sparse_data: Optional[Dict[str, onnx.SparseTensorProto]] = None,
+    subgraph_opset_imports: Optional[List[onnx.OperatorSetIdProto]] = None,
```

Comment:
minor nit: I think `opset_imports` and `ir_version` might be sufficient as the parameter names. I realize they are used only for subgraphs, but it is unlikely to cause any confusion or ambiguity to use the shorter name. It will be easier for callers, when they use keyword-arguments.

Reply:
I was in the debate of `opset_imports` and `subgraph_opset_imports`, but yes I agreed using shorter name here is cleaner. Just updated. Will merge this PR soon. Thank you for bringing this up!